examples/java/pom.xml

```diff
@@ -194,6 +198,24 @@
           </execution>
         </executions>
       </plugin>
+
+      <!-- Override parent configuration.
+           Spark/Flink runner dependencies are necessary to be in shaded jar
+           in order to run WordCountIT with Spark/Flink runner -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>2.10</version>
```

Comment:
Yes, that is caused by them being compile dependencies. If you make them `runtime` it will be fine. Or, if you do reference them in the code, mark them `optional`.